apps/public-reference/pages/applications/contact/alternate-contact-contact.tsx

```diff
@@ -46,19 +46,14 @@ export default () => {
     <FormsLayout>
       <FormCard header={listing?.name}>
         <ProgressNav
```

Comment:
Could the progress nav, useFormConductor, back/next buttons etc all live in a shared applicationstep container component and the actual view rendering logic lives in unique components for each step? The only challenge I don't have an answer to right now is where the data setting logic in onSubmit should live...

Just thinking about how difficult it will be to test each of these step views, since we're mixing state connection and display logic

Reply:
Yeah it's been an uphill battle working on DRYing up the various form pages, but I think once we have the conductor step stuff fully in place, we can take most of the form saving logic out of there as well as beef up a dedicated shared layout (so for example the back link is pulled in once). Probably a bit much to chew off for this PR though.

Reply:
Sounds good to me!